### ISSUE TITLE
Если по дизайну, вы храните корабли в игроке, то логичнее там же и инициализировать

### ISSUE THREAD

**Issue by ahvastunov-itmo** [egoarka/BattleShip-c#11]
https://github.com/egoarka/BattleShip-c/blob/f5be01a825ca248c5343facab0acba9a5232720b/world.c#L33

**Comment by egoarka** [comment]
Да, у меня так изначально и было, но потом сделал как есть, чтобы сразу ссылки на вражеские и собственные лодки ссылки указать в одном месте.
